examples/image_streamer/plan_scripts.py

```diff
@@ -72,6 +72,13 @@
     print(plan_script_item['name'])
 print("***** done *****\n")
 
+# Get used by and read only
+print("Gets builds plans which uses a particular read only plan script")
+build_plans = image_streamer_client.plan_scripts.get_usedby_and_readonly("cbfd487e-3d92-4eb4-b877-f9ea7c16a271")
```

Comment:
Instead of hard coding here, we can either declare a variable in the beginning of this script or we can extract id from uri like below:
id_plan_script_by_id = extract_id_from_uri(plan_scripts['uri'])